### ISSUE TITLE
Magnetron code test

### ISSUE THREAD

**Issue by drphilmarshall** [eggplantbren/TimeBombs#15]
Hey @rumbaugh, Brendon merged in a bunch of new code in [this commit](https://github.com/eggplantbren/TimeBombs/commit/cb7b54a3f98f395222bd5332fa6383d49929ec47). Can you git pull please? This will break your test run, so you'll need to make it work again :-)

**Comment by eggplantbren** [comment]
The noticable changes will be i) It will run a little bit slower (sorry) ii) a different number of output columns. Let me know if you encounter any problems.

**Comment by rumbaugh** [comment]
I put this off because it would mess up my runs, but I went and pulled the new version. I tried doing a run, but it appears to have the same number of output columns as before. How many should it have had?

**Comment by rumbaugh** [comment]
Okay, I realized I hadn't recompiled the run script. Now, when I try to do a TimeBombs run, it gives me a segmentation fault right after it starts. The output is like this: 

Found 110 points in file /u/ki/nrumbaug/Fermi/data/test/testlightcurve_50.tau_16.31.mu_0.625.5.13.15.dat.
testing...
Using 16 threads.
Target compression factor between levels = 2.7182818284590451
Seeding random number generator with 1438816184.
1 100000 50000 2000 200 4 100 1000 /u/ki/nrumbaug/Fermi/TimeBombs/output/sample.testlightcurve_50.iter_0.7.1.15.dat
Segmentation fault
